content/zh/docs/ops/diagnostic-tools/cni/index.md

```diff
@@ -78,6 +78,6 @@ $ kubectl logs POD_NAME -n POD_NAMESPACE -c istio-validation
 
 CNI 插件出现故障的另一个症状是，在启动时，应用程序 Pod 不断被逐出。
 这通常是因为插件没有正确安装，因此无法设置 Pod 流量重定向。
-CNI [race repair logic](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)
-认为 Pod 由于竞争条件而损坏，并连续逐出 Pod。遇到此问题时，请检查 CNI DaemonSet 日志，
+CNI 的[竞态修复逻辑](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)
```

Comment:
```suggestion
CNI 的[竞争条件和缓解措施逻辑](/zh/docs/setup/additional-setup/cni/#race-condition-mitigation)
```

Reply:
yes, follow the https://istio.io/latest/zh/docs/setup/additional-setup/cni/#race-condition-mitigation